string eef_name
float64[] trajectory
string[] gripper_values
bool feedback
